### DISCUSSION TITLE
Links to anchors show the anchor name instead of the heading title

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#529] [0. General bugs and questions]
I had posted this as a comment on a previous post about the redesign, but bumping it up to a separate post since this is really more of a bug than a display preference.

When you add a direct link to an anchor within a topic, in the published site the link displays the anchor name (name-of-the-anchor) instead of the text of the heading (Title of the heading).

For a link to another heading, the link text is **#name-of-the-anchor**.

For a link to a different topic, the link text is **Title of the topic#name-of-the-anchor**, which looks especially strange.

![LinkToAnchor](https://github.com/GitbookIO/community/assets/107077736/28df8c2d-7817-4091-acc1-92ed63bd6c57)

I always rename the anchors to use a specific format, which is fine for a URL but not intended to be read in text. Before the redesign links to anchors displayed the heading name.

**Comment by addisonschultz** [comment]
Thanks @JaniceManwiller - we're be working on this one asap!

**Comment by SamyPesse** [comment]
This one was fixed a few days ago. Let us know otherwise.
